src/main/java/org/opensearch/security/dlic/rest/api/MultiTenancyConfigApiAction.java

```diff
@@ -177,6 +183,12 @@ private void updateAndValidatesValues(final ConfigV7 config, final JsonNode json
         if (Objects.nonNull(jsonContent.findValue(MULTITENANCY_ENABLED_JSON_PROPERTY))) {
             config.dynamic.kibana.multitenancy_enabled = jsonContent.findValue(MULTITENANCY_ENABLED_JSON_PROPERTY).asBoolean();
         }
+        if (jsonContent.hasNonNull(DASHBOARD_SIGNIN_OPTIONS) && jsonContent.findValue(DASHBOARD_SIGNIN_OPTIONS).isEmpty() == false) {
+            JsonNode newOptions = jsonContent.findValue(DASHBOARD_SIGNIN_OPTIONS);
+            List<DashboardSignInOption> options = getNewDashboardSignInOptions(newOptions);
+            config.dynamic.kibana.dashboardSignInOptions = options;
```

Comment:
The standard for the security config is lower_snake_case. Can this be updated to `dashboards_sign_in_options` or shortened to `sign_in_options`.

This PR looks good to me otherwise!